src/views/Shipment.vue

```diff
@@ -0,0 +1,379 @@
+        </section>
+      </main>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import {
+  IonBackButton,
+  IonBadge,
+  IonButton,
+  IonButtons,
+  IonCard,
+  IonCardHeader,
+  IonCardTitle,
+  IonCheckbox,
+  IonChip,
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonNote,
+  IonPage,
+  IonSelect,
+  IonSelectOption,
+  IonThumbnail,
+  IonTitle,
+  IonToolbar,
+  popoverController
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import {
+  addOutline,
+  businessOutline,
+  calendarOutline,
+  checkmarkDoneOutline,
+  downloadOutline,
+  ellipsisVerticalOutline,
+  informationCircleOutline,
+  locationOutline,
+  openOutline,
+  sendOutline,
+  shirtOutline,
+  syncOutline,
+  ticketOutline,
+  timeOutline
+} from 'ionicons/icons';
+import Image from '@/components/Image.vue';
+import EditShippedQuantityPopover from '@/components/EditShippedQuantityPopover.vue';
+
+export default defineComponent({
+  name: 'Shipment',
+  components: {
+    IonBackButton,
+    IonBadge,
+    IonButton,
+    IonButtons,
+    IonCard,
+    IonCardHeader,
+    IonCardTitle,
+    IonCheckbox,
+    IonChip,
+    IonContent,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonNote,
+    IonPage,
+    IonSelect,
+    IonSelectOption,
+    IonThumbnail,
+    IonTitle,
+    IonToolbar,
+    Image
+  },
+  data() {
+    return {
+      status: "Shipped" // default value
+    }
+  },
+  methods: {
+    changeStatus (ev: any) {
+      this.status = ev['detail'].value
+    },
+    async openShippedPopover(ev: Event) {
+      const popover = await popoverController.create({
+        component: EditShippedQuantityPopover,
+        event: ev,
+        showBackdrop: false,
+        translucent: true,
+      });
+      return popover.present();
+    },
+  },
+  setup() {
+    return {
+      addOutline,
+      businessOutline,
+      checkmarkDoneOutline,
+      calendarOutline,
+      downloadOutline,
+      ellipsisVerticalOutline,
+      informationCircleOutline,
+      locationOutline,
+      openOutline,
+      sendOutline,
+      shirtOutline,
+      syncOutline,
+      ticketOutline,
+      timeOutline
+    }
+  }
+});
+</script>
+
+<style scoped>
+/* To hide selected text which appear after selecting any option*/
+ion-select::part(text) {
+  display: none;
+}
+/* To remove margin between badge and ion-select */
+ion-select {
+  margin-inline-start: 0;
+}
+
+.list-item {
+  --columns-mobile: 4;
+}
+
+.mobile {
+  display: unset;
+}
+
+.product-header {
+    display: flex;
+    justify-content: space-between;
+}
+
+@media (min-width: 991px) {
+  .list-item {
+    --columns-desktop: 5;
+  }
+
+  .product-detail {
+    display: grid;
+    grid-template-columns: 250px 1fr;
+    align-items: center;
+  }
+  .product-image {
+    border: 1px solid var(--ion-color-medium);
+    border-radius: 10px;
+  }
+
+  .product-info {
```

Comment:
where is this class being used?

Reply:
I created this styling for margin between product image and its info. I have added class to the markup.